docs/user_manual/home_screen_widgets.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "26. Home Screen Widgets"
+section: 26
+---
+
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Search" %}
+
+The Search widget can be used to quickly find an item in MISO. This includes samples, libraries, library dilutions,
```

Comment:
I think it's worth mentioning that projects and runs can't be searched from this box.

Reply:
Added note about other widgets for those